### PR TITLE
Update terminology and style BancorNetwork

### PR DESCRIPTION
**Pull request by lbeder**

### CODE REVIEW

File: solidity/contracts/converter/interfaces/IConverter.sol [hunk 1]
```diff
@@ -93,16 +93,16 @@ interface IConverter is IOwned {
...
      * @param conversionFee conversion fee in units of the target token
      */
     event Conversion(
-        IReserveToken indexed fromToken,
-        IReserveToken indexed toToken,
+        IReserveToken indexed sourceToken,
+        IReserveToken indexed targetToken,
         address indexed trader,
         uint256 amount,
         uint256 returnAmount,
```
Comment (barakman) [inline solidity/contracts/converter/interfaces/IConverter.sol:108]: Should also rename `amount` to `sourceAmount` and `returnAmount` to `targetAmount` IMO.
Another option is `inputAmount` and `outputAmount`.
It should then also be applied in function `dispatchConversionEvent` in the standard-pool converter.

File: solidity/contracts/BancorNetwork.sol [hunk 2]
```diff
@@ -64,20 +64,20 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient, R
...
-        IReserveToken indexed _toToken,
-        uint256 _fromAmount,
-        uint256 _toAmount,
-        address _trader
+        IConverterAnchor indexed anchor,
+        IReserveToken indexed sourceToken,
+        IReserveToken indexed targetToken,
+        uint256 sourceAmount,
+        uint256 returnAmount,
+        address trader
```
Comment (barakman) [inline solidity/contracts/BancorNetwork.sol:80]: Should also rename `returnAmount` to `targetAmount` IMO.

File: solidity/contracts/converter/interfaces/IConverter.sol (hunk 1 above)
Comment (barakman) [reply solidity/contracts/converter/interfaces/IConverter.sol:108]: Also, in function `doConversion` in the standard-pool converter, we are still using local variables named `fromAmount` and `toAmount `, which we probably want to rename to `sourceAmount` and `targetAmount` as part of this task.

**Review by barakman — CHANGES_REQUESTED** [review]

File: solidity/test/StandardPoolConverter.js [hunk 3]
```diff
@@ -387,11 +387,11 @@ describe('StandardPoolConverter', () => {
...
-                        _fromToken: reserveToken1.address,
-                        _toToken: reserveToken2.address,
-                        _fromAmount: amount,
-                        _toAmount: purchaseAmount
+                        anchor: poolToken.address,
+                        sourceToken: reserveToken1.address,
+                        targetToken: reserveToken2.address,
+                        sourceAmount: amount,
+                        returnAmount: purchaseAmount
                     });
```
Comment (barakman) [inline solidity/test/StandardPoolConverter.js:395]: If you take my suggestion and rename `returnAmount` in the contract, then don't forget to rename it here too.

**Review by barakman — CHANGES_REQUESTED** [review]

File: solidity/contracts/converter/interfaces/IConverter.sol (hunk 1 above)
Comment (lbeder) [reply solidity/contracts/converter/interfaces/IConverter.sol:108]: 👍

File: solidity/contracts/BancorNetwork.sol (hunk 2 above)
Comment (yudilevi) [reply solidity/contracts/BancorNetwork.sol:80]: +1

**Review by yudilevi — COMMENTED** [review]
no further comments other than barak's.

**Review by yudilevi — APPROVED** [review]

**Review by barakman — APPROVED** [review]

**Pull request merged**